ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -586,6 +586,37 @@ tests:
     test:
     - ref: fips-check-optional-operator-images-scan
     workflow: cucushift-installer-rehearse-aws-ipi-proxy
+- as: aws-ipi-proxy-fips-f360-compliance-operator
+  cron: 25 19 16 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FILTERS_ADDITIONAL: ""
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~HyperShiftMGMT&;~Disruptive&
+      TEST_PARALLEL: "3"
+      TEST_SCENARIOS: Compliance_Operator
+    test:
+    - ref: openshift-extended-test
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy
+- as: aws-ipi-proxy-fips-f360-compliance-operator-destru
+  cron: 32 3 16 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@complianceoperator'
+      FILTERS_ADDITIONAL: ""
+      FIPS_ENABLED: "true"
+      TEST_FILTERS: ~HyperShiftMGMT&;Disruptive&
+      TEST_SCENARIOS: Compliance_Operator
+      TEST_TIMEOUT: "40"
+    test:
+    - ref: etcd-encryption
```

Comment:
I am not sure if it is deserved to introduce a dedicated test chain for `compliance-operator` testing, cc @liangxia for more insight.

Reply:
There are several reasons I would prefer to have separate release jobs for compliance-operator:
1. For compliance-operator, there are 127 test cases automated,a lot of test cases are destructive and long duration(the test run will cost more than 5 hours). 
2. For compliance operator test cases, a lot of them only support amd64 and cpaas only. Thus there are a lot of test cases don't have chance to run if Jenkins jobs get deprecated.
3. The release frequency of compliance operator is monthly. Releasing compliance operator will be a hard work if Jenkins jobs get deprecated and no other jobs to trigger the test.
Thus I would prefer to have test jobs for it.